src/Service.Tests/SqlTests/SqlTestHelper.cs

```diff
@@ -394,6 +455,63 @@ public static string ConstructGQLTypenameResponseNTimes(string typename, int tim
             return typenameResponseBuilder.ToString();
         }
 
+        /// <summary>
+        /// Compares the expected JSON result with the actual JSON result after applying necessary transformations.
+        /// </summary>
+        /// <param name="expectedJson">The expected JSON string.</param>
+        /// <param name="actualJson">The actual JSON string to compare.</param>
+        public static void PerformTestEqualJsonStringsWithTransformations(string expectedJson, string actualJson)
+        {
+            // Parse the expected and actual JSON
+            JsonDocument expectedDocument = JsonDocument.Parse(expectedJson);
+
+            // Transform expected JSON to match the actual GraphQL format
+            string transformedExpectedJson = TransformExpectedJson(expectedDocument);
+
+            // Compare the transformed expected JSON with the actual JSON
+            Assert.AreEqual(transformedExpectedJson, actualJson, "The JSON results do not match.");
+        }
+
+        /// <summary>
+        /// Transforms the expected JSON to match the format of the actual GraphQL output.
+        /// This method is generic and can handle various structures.
+        /// </summary>
+        /// <param name="expectedDocument">The expected JSON document.</param>
+        /// <returns>The transformed expected JSON as a string.</returns>
+        private static string TransformExpectedJson(JsonDocument expectedDocument)
+        {
+            // Create a new JSON array to hold the transformed data
+            JsonArray transformedArray = new();
+
+            // Iterate through each element in the expected JSON array
+            foreach (JsonElement element in expectedDocument.RootElement.EnumerateArray())
+            {
+                JsonObject transformedElement = new();
+
+                // Iterate through each property in the element
+                foreach (JsonProperty property in element.EnumerateObject())
+                {
+                    // Generic transformation logic
+                    // For example, you might want to rename properties or adjust their values
+                    string newPropertyName = property.Name;
+
+                    // Example transformation: if the property name starts with "max_", remove the prefix
+                    if (newPropertyName.StartsWith("max_"))
```

Comment:
curious why do we need this transformation? does SQL preprend max_ to some column names which represent max operations and GraphQL doesn't?